src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Color DEFAULT_COLOR = Color.WHITE;
+    private static final int DEFAULT_RADIUS = 10;
+    private static final int NUMBER_OF_FIGURES = 5;
+    private static final int MAX_VALUE = 20;
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int randomFigureNumber = random.nextInt(NUMBER_OF_FIGURES) + 1;
+        switch (randomFigureNumber) {
+            case 1:
+                return getRandomCircle();
+            case 2:
+                return getRandomIsoscelesTrapezoid();
+            case 3:
+                return getRandomRectangle();
+            case 4:
+                return getRandomRightTriangle();
+            case 5:
+                return getRandomSquare();
+            default:
+                return null;
```

Comment:
it is recommended to not return null in the default case, better to make case 5 default instead